db/migrations/9.4.04.sql

```diff
@@ -0,0 +1,247 @@
+DO $$
+BEGIN
+   IF EXISTS (
+      SELECT 1
+      FROM information_schema.columns 
+      WHERE table_schema = 'public'
+            AND table_name = 'accounts'
+            AND column_name = 'identity_id'
+      ) 
+   THEN 
+      IF EXISTS (
+         SELECT 1
+         FROM information_schema.columns 
+         WHERE table_schema = 'public'
+               AND table_name = 'accounts'
+               AND column_name = 'identity_id'
+               AND is_nullable = 'NO' )
+      THEN
+         ALTER TABLE public.accounts ALTER COLUMN identity_id DROP NOT NULL;
+      END IF;
+   END IF;
+END $$;
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS account_history (
+    id text NOT NULL DEFAULT uuid_generate_v4(),
+    account_id text,
+    identity_id text,
+    event_id public_enum_8f5a39c8ee,
+    permissions_id text,
+    datetime timestamp(6),
+    updated_block_id text
+);
```

Comment:
We always added `created_block_id`, `updated_block_id` to all the entities, and we don't use uuid generators for filling in the PK sequence (because SQ doesn't follow this). Idea is to have the same table structure as the SQ does. 

Also, you forgot to add constraints and indexes

What I generally do is run an SQ instance with the new table added and then check the table structure using `\d+ tablename` . This gives me the exact idea on which constraints I need to add, which indexes etc. etc